src/NativeShell/src/api/ApiClientBridge.java

```diff
@@ -0,0 +1,21 @@
+package com.mb.android.api;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.mb.android.logging.AppLogger;
+
+import mediabrowser.apiinteraction.android.AndroidCredentialProvider;
+import mediabrowser.apiinteraction.android.VolleyHttpClient;
+import mediabrowser.model.logging.ILogger;
+import mediabrowser.model.serialization.IJsonSerializer;
+import mediabrowser.model.session.ClientCapabilities;
+
+public class ApiClientBridge {
+    public VolleyHttpClient httpClient;
+
+    public ApiClientBridge(Context context) {
+        ILogger logger = AppLogger.getLogger(context);
+        httpClient = new VolleyHttpClient(logger, context);
```

Comment:
if the sole purpose of this is add logging, can we drop this file altogether?..

Reply:
Not quite yet, although this should happen eventually. It's still required for the java api client for now.